src/containers/LazyLoader/loader.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+
+const Loader = () => {
+  return (
+    <SkeletonTheme color="#313236" highlightColor="#49494D">
+      <div className="skeleton-header">
+        <span className="skeleton__menu-bar">
```

Comment:
as we are following the BEM, should it be named as `skeleton-header__menu-bar`?

Reply:
Fixed in [6c44c1b](https://github.com/rafayet-monon/surveyor/pull/9/commits/6c44c1bd55bc12370c81af6d4a2cf087d87dc30f)